tests/scripts/test_ray_serve_1.py

```diff
@@ -0,0 +1,17 @@
+import ray
+import sys
+import ray.serve as serve
+import os
+import requests
+from ray._private.test_utils import wait_for_condition
+
+ray.init(address='ray://127.0.0.1:10001', namespace=sys.argv[1])
+serve.start(detached=True)
+
+@serve.deployment
+def d(*args):
+    return "HelloWorld"
+d.deploy()
+val = ray.get(d.get_handle().remote())
```

Comment:
Based on the [Ray Serve API doc](https://docs.ray.io/en/latest/serve/package-ref.html), `get_handle()` is deprecated and may be removed in future Ray releases. Maybe we need to use other functions.